javascript-bank/bank.js

```diff
@@ -0,0 +1,44 @@
+/* exported Bank */
+
+function Bank() {
+  this.nextAccountNumber = 1;
+  this.accounts = [];
+}
+
+Bank.prototype.openAccount = function (holder, balance) {
+  if (Number.isInteger(balance) && balance > 0) {
+    var newAccount = new Account(this.nextAccountNumber, holder);
+    newAccount.deposit(balance);
+    this.accounts.push(newAccount);
+    this.nextAccountNumber++;
+    return newAccount.number;
+  } else {
+    return null;
+  }
+};
+
+Bank.prototype.getAccount = function (number) {
+  var counter = 0;
+  for (var i = 0; i < this.accounts.length; i++) {
+    if (this.accounts[i].number === number) {
+      return this.accounts[i];
+    } else {
+      counter++;
+    }
+  }
+  if (counter === this.accounts.length) {
+    return null;
+  }
```

Comment:
You can just `return null`, no need for the counter variable. If your code reaches this point, that means the loop has ended and no account was found matching the number given.